pints/_mcmc/_global_adaptive_covariance.py

```diff
@@ -0,0 +1,62 @@
+#
+# Base class for Adaptive covariance MCMC methods
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2018, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+
+
+class GlobalAdaptiveCovarianceMCMC(pints.AdaptiveCovarianceMCMC):
+    """
+    Base class for single chain MCMC methods that globally adapt a
+    proposal covariance matrix when running, in order to control the
+    acceptance rate.
+
+    *Extends:* :class:`AdaptiveCovarianceMCMC`
+    """
+    def tell(self, fx):
```

Comment:
I can't really tell from this code where the implementing classes get to do their thing. Can that happen in some appropriately named method that's called by this class?

E.g. self._adapt_log_lambda ?

Reply:
I could, but feel that, before implementing the other AC methods, I'd be a bit unsure whether self._adapt_log_lambda is needed? Any chance we could wait with this then update later if needed/

Reply:
Sure!